CRS/Cirrus.Node/Controllers/IndexerContractsController.cs

```diff
@@ -79,11 +83,76 @@ public IActionResult GetContractInfo([FromQuery] string txHash)
                 logResponses = deserializer.MapLogResponses(receipt.Logs);
             }
 
-            return this.Json(new ContractReceiptResponse(receipt, logResponses ?? new List<LogResponse>(), this.network)
+            EnrischLogs(receipt, typeName, logResponses);
+
+            return this.Json(new ContractReceiptResponse(receipt, logResponses, this.network)
             {
                 ContractCodeType = typeName,
             });
         }
 
+        private void EnrischLogs(Receipt receipt, string typeName, List<LogResponse> logResponses)
+        {
+            if (typeName == "StandardToken")
```

Comment:
Please make an static variable or enum for these, so you can do "ContractTypeName.StandardToken", etc.